tests/pyseir_end_to_end_test.py

```diff
@@ -16,10 +16,10 @@
 # (https://stackoverflow.com/questions/27476642/matplotlib-get-rid-of-max-open-warning-output)
 @pytest.mark.filterwarnings("error", "ignore::RuntimeWarning")
```

Comment:
If there's significance to this being Idaho I can revert this, but I updated this just so that the tests would stop failing

Reply:
Thank you!! 🙏  I was meaning to do that but hadn't gotten to it.